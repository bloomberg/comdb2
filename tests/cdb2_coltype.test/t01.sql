CREATE TABLE t1(x1 SMALLINT, x2 BIGINT, x3 INT, x4 INTEGER, x5 LARGEINT);$$
CREATE TABLE t2(x1 SMALLFLOAT, x2 FLOAT, x3 DOUBLE);$$
CREATE TABLE t3(x1 CHAR(2), x2 VARCHAR(2), x3 BLOB);$$
INSERT INTO t1 (x1, x2, x3, x4, x5) VALUES (NULL, NULL, NULL, NULL, NULL);
INSERT INTO t1 (x1, x2, x3, x4, x5) VALUES (0, 1, 2, 3, 4);
INSERT INTO t2 (x1, x2, x3) VALUES (NULL, NULL, NULL);
INSERT INTO t2 (x1, x2, x3) VALUES (-1.0, 0.0, 1.0);
INSERT INTO t3 (x1, x2, x3) VALUES (NULL, NULL, NULL);
INSERT INTO t3 (x1, x2, x3) VALUES ('1', '12', x'616263');
SELECT x1, x2, x3, x4, x5 FROM t1 ORDER BY x1
SELECT x1, x2, x3 FROM t2 ORDER BY x1
SELECT x1, x2, x3 FROM t3 ORDER BY x1
