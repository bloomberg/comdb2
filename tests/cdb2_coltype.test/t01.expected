(INTEGER rows inserted=1)
(INTEGER rows inserted=1)
(INTEGER rows inserted=1)
(INTEGER rows inserted=1)
(INTEGER rows inserted=1)
(INTEGER rows inserted=1)
(NULL x1=NULL, NULL x2=NULL, NULL x3=NULL, NULL x4=NULL, NULL x5=NULL)
(INTEGER x1=0, INTEGER x2=1, INTEGER x3=2, INTEGER x4=3, INTEGER x5=4)
(NULL x1=NULL, NULL x2=NULL, NULL x3=NULL)
(REAL x1=-1.000000, REAL x2=0.000000, REAL x3=1.000000)
(NULL x1=NULL, NULL x2=NULL, NULL x3=NULL)
(CSTRING x1='1', CSTRING x2='12', BLOB x3=x'616263')
